config_defaults.yml

```diff
@@ -84,6 +84,8 @@
   branches:
     - master
     - /^v\d/
+  use_litmus:
+    enabled: true
```

Comment:
Doesn't seem like we can make this the default yet? (absent a major version bump)

Reply:
Agreed. This is a blocker to merge.

Reply:
hey @scotje @glennsarti 
I don't understand. Sorry :(  Why do you consider this a blocker?
 If we add it is as disabled, it's still a problem? We can enable it just for modules which are already converted. Does it make any sense?

Reply:
I disabled it form config_defaults and now the current functionality is not affected. Probably was a bad idea enabling it

Reply:
👍  Yeah I just meant that we can't make Litmus the default for people upgrading their existing modules to the latest template version.

Reply:
hey @scotje ,

Updated to use_litmus: true. 
By default it's set to false. 
Also you can specify some sub-keys used for testing:
-     provision_list: [list1, list2]
-     puppet_collection: [puppet6]
-     rvm: 2.4.4
-     install_wget: yes